outline.md

```diff
@@ -0,0 +1,59 @@
+You're the dedicated champion for inner source in your company.
+What do you do now?
+How do you realistically affect the behavior of dozens or hundreds of teams to the point where robust and pervasive inner sourcing is a normal part of the way that engineering is done?
+
+This presentation shares principles, practical anecdotes, and relatable examples on this situation gleaned from experience at Nike.
+While not claiming all of the answers, it is insightful summary of over a year of work in the area.
+
+## Audience
+This presentation will be given at [the ISC Spring Summit](http://paypal.github.io/InnerSourceCommons/events/isc-spring-2018-agenda/).
+It is intended for those that are evangelizing inner source in a specific company.
+
+## Detailed description
+
+#### Outline
+
+An inner source program is the set of tools, documentation, training, and so forth provided so that any team can participate in inner source effectively as either a maintainer or contributor.
+To frame the talk, we'll explain the goals of an inner source program as well as the context within-which we've begun to pursue those goals.
+Even though the learning in this presentation is occurring in that context, we suspect that much of what's here applies generally.
+
+When building an inner source program you should have a vision of what services you will provide to facilitate inner sourcing _as well as_ a strategy you will employ to roll out and make effective these services in your organization.
+We'll explain principles, resources, and specific examples that can aid and guide in doing so.
+
+The services of the inner source program should provide two things.
+First, they must teach, define, and document the type of culture that is conducive to inner source.
+Second, there must be software that reinforces that culture while simultaneously lowering the barrier to entry to participating in that culture.
+Examples of these services include:
+
+* Documentation on how to do inner source (and relation of that documentation to [InnerSourcePatterns](http://innersourcecommons.org/patterns)).
+* Software for automated, two-way discovery of inner source projects (maintainer to contributor/user and contributor/user to maintainer).
+* Software for a data-driven understanding of the impact of inner source projects.
+* Software/convention to facilitate individuals communicating with one another.
+* Curated training on the above software and documentation.
+
+The rollout of an inner source program is similar to the rollout of any large, grass-roots initiative.
+Documented patterns in this area (well-summarized in [the Fearless Change series](http://www.fearlesschangepatterns.com/)) apply and work.
+We'll review real examples of how these patterns played out in our journey.
+We'll also share a few patterns unique to inner source rollout including:
+
+* Run your inner source program as inner source projects.
+* Do (or say) nothing save it is documented.
```

Comment:
Not sure what this means.

Reply:
We write (or code) down what we are telling others to do and what we are doing ourselves.

Reply:
The principle is more about teaching a developer to fish so they can feed themselves rather than fishing for them.

Reply:
I think we are always teaching people to fish.  This advice is even higher up.  More like:

* Create a fishing school rather than always teaching people yourself.
* Build a fishing machine so people don't even need to fish.

Reply:
oh even better articulation!! Nice!

Reply:
Looking again at what I'm proposing an inner source program should provide (documentation and software), you could really say that the inner source program should provide a fishing school and fishing machines.  Move above that level of teaching others to fish one-on-one.  Maybe this mode of explanation and analogy could be woven into the whole talk?

Reply:
Maybe that's also a way of speaking against the top-down approach of getting management to "make" people do inner source.  Is doing so like fishing for them?  They should be up-leveled to fish for themselves.

And not only that, the way that you up-level them to fish for themselves is by having them attend your fishing school and use your fishing machines.  We're takin' fishing from the 19th century to the 21st century.

Does that make sense?  It was helpful for me to write it, at least.

Reply:
yea I think it would be a nice way to frame the whole conversation and keep it relatable.

Reply:
OK, great.